src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -62,24 +101,41 @@ def assert_sphinx_is_available(logger):
 def sphinx_generate(project, logger):
     """Runs sphinx-build against rst sources for the given project.
     """
-    logger.info("Running sphinx-build")
-
-    log_file = project.expand_path("$dir_target/reports/{0}".format("sphinx-build"))
+    task_name = getattr(sphinx_generate, NAME_ATTRIBUTE)
     build_command = get_sphinx_build_command(project)
+    run_build(build_command, task_name, logger, project)
 
-    if project.get_property("verbose"):
-        logger.info(build_command)
 
-    exit_code = execute_command(build_command, log_file, shell=True)
-    if exit_code != 0:
-        raise BuildFailedException("Sphinx build command failed. See %s for details.", log_file)
+@task("sphinx_quickstart", "starts a new sphinx project")
+@depends("prepare")
+def sphinx_quickstart_generate(project, logger):
+    """Runs sphinx-build against rst sources for the given project.
+    """
+    task_name = getattr(sphinx_quickstart_generate, NAME_ATTRIBUTE)
+    build_command = get_sphinx_quickstart_command(project)
+    run_build(build_command, task_name, logger, project)
+
+
+def get_sphinx_quickstart_command(project):
+    """Builds the sphinx-quickstart command using project properties.
+        :param -q: Quiet mode that will skips interactive wizard to specify options.
+        :param -p: Project name will be set.
+        :param -a: Author names.
+        :param -v: Version of project.
+    """
+    options = ["-q",
+               "-p %s" % project.name,
+               "-a %s" % project.doc_author,
```

Comment:
I think we could drop the `doc_author` entirely and simply use the `project.author` array (I can't think of any case where the doc authors are not a subset of the project authors)